### ISSUE TITLE
Make file

### ISSUE THREAD

**Issue by james3d** [jcbtmy01/Toomey_CSCI2270_FinalProject#1]
Hey guys you forgot to include a make file, so I don't really know what to do. I'm not sure if you meant to put something on or am I just supposed to compile all the .cpp files.

**Comment by jcbtmy01** [comment]
yeah sorry, we ran out of time to make one. just download the repository and copy 
g++ main.cpp organizer.cpp BinaryTree.cpp HashTable.cpp LinkedList.cpp RB.cpp -o RunTime

**Comment by james3d** [comment]
Just ran that and I got a bunch of errors.

**Comment by jcbtmy01** [comment]
shit my bad, alright now it should work, i guess it didnt get our last commit.

**Comment by james3d** [comment]
Sick, this is a sweet project btw.
